x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_exploration/components/classification_exploration/evaluate_panel.tsx

```diff
@@ -293,16 +294,7 @@ export const EvaluatePanel: FC<EvaluatePanelProps> = ({ jobConfig, jobStatus, se
                         <span>{getHelpText(dataSubsetTitle)}</span>
                       </EuiTitle>
                       <EuiFlexItem grow={false}>
-                        <EuiIconTip
-                          anchorClassName="mlDataFrameAnalyticsClassificationInfoTooltip"
-                          content={i18n.translate(
-                            'xpack.ml.dataframe.analytics.classificationExploration.confusionMatrixTooltip',
-                            {
-                              defaultMessage:
-                                'The multi-class confusion matrix contains the number of occurrences where the analysis classified data points correctly with their actual class as well as the number of occurrences where it misclassified them with another class',
-                            }
-                          )}
-                        />
+                        <MulticlassConfusionMatrixHelpPopover />
```

Comment:
Can you add `alignItems="center"` to the parent `EuiFlexGroup`

```
<EuiFlexGroup gutterSize="none" alignItems="center">
```

as I think that gives better alignment of the icon with the panel title:

![image](https://user-images.githubusercontent.com/7405507/121518367-4d375680-c9e8-11eb-849e-06eee7be412b.png)